components/collections/Cargo.toml

```diff
@@ -36,7 +36,7 @@ serde = { version = "1.0", default-features = false, features = ["derive", "allo
 databake = { version = "0.1.3", path = "../../utils/databake", features = ["derive"], optional = true }
 
 [dev-dependencies]
-postcard = { version = "1.0.0", features = ["alloc"] }
+postcard = { version = "1.0.0", features = ["alloc"], default_features = false }
```

Comment:
I feel the same about `no-default-features` on `postcard` as I do on `serde_json`